### PR TITLE
add committee of 2018-19 and the current committee of 2019-20

### LINKED ISSUE #321: /api/committee not retrieving the latest committee
### Expected behaviour

Retrieves the latest committee members. Though not updated to reflect the incoming 2019-20 it should at lest show the 2018-19 committee.

### Actual behaviour

Retrieves the committee members of the 2017-28 committee.

### Screenshot of the problem (if applicable)

image

### Checklist

*I confirm that before I submitted this issue I did the following:*

- [x] Cleared my cookies.
- [x] Cleared my cache.
- [x] Refreshed the page.

### PR DESCRIPTION
**Pull request by theycallmemac**
Closes #321 

### Description
add committee of 2018-19 and the current committee of 2019-20

### Checklist

*I confirm that before I submitted this pull request I did the following:*

- [x] Confirmed that no other open PRs address this issue.
- [x] Checked out the latest version of the site.
- [x] Ran `yarn start` and confirmed that there were no issues.
- [ ] Annotated my code with the relevant license information (if applicable).

If someone wants to make sure this is correct that would be great.

### CODE REVIEW

File: source/about/past-committees/the-committee-2018-2019/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - position: Treasurer
+    name: Ciara Godwin
+    nick: hexagon
+    image: hexagon.jpg
+  - position: Public Relations Officer
+    name: Josh Malone
+    nick: sangreal
+    image: sangreal.jpg
+  - position: Graphics Design Officer
+    name: Theo Morgan Coyne
```
Comment (cliodhnaharrison) [inline source/about/past-committees/the-committee-2018-2019/index.md]: Theo Morgan Coyne -> Theo Coyne Morgan

 At least that's his name on Facebook

File: source/about/past-committees/the-committee-2019-2020/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - position: Treasurer
+    name: Sean Hammond
+    nick: verren
+    image: verren.jpg
+  - position: Public Relations Officer
+    name: Tara Collins
+    nick: tara
+    image: tara.jpg
+  - position: Graphics Design Officer
+    name: Theo Morgan Coyne
```
Comment (cliodhnaharrison) [inline source/about/past-committees/the-committee-2019-2020/index.md]: same again

File: source/about/past-committees/the-committee-2018-2019/index.md (hunk 1 above)
Comment (theycallmemac) [reply source/about/past-committees/the-committee-2018-2019/index.md]: D:

**Comment by theycallmemac** [comment]
Should be good now

**Review by cliodhnaharrison — APPROVED** [review]

**Review by Sangreal8 — APPROVED** [review]

**Review by benmcmahon100 — APPROVED** [review]

**Pull request merged**